lib/utils/utils.js

```diff
@@ -213,12 +213,22 @@ function getSourceModuleNameForIdentifier(context, node) {
   const importDeclaration = program.body
     .filter(isImportDeclaration)
     .find(importDeclaration =>
-      importDeclaration.specifiers.some(specifier => specifier.local.name === node.name)
+      importDeclaration.specifiers.some(
+        specifier => specifier.local.name === getSourceModuleName(node)
+      )
     );
 
   return importDeclaration ? importDeclaration.source.value : undefined;
 }
 
+function getSourceModuleName(node) {
+  if (node.type === 'MemberExpression') {
+    return node.object.name;
```

Comment:
This is assuming that object is an identifier which is the case with `DS.Model` but not with longer paths like `Some.Long.Chained.Path`. You need to check the type of the node.object here and add a loop/recursion to handle longer paths.